src/ServiceInsight.Tests/MessageBodyViewModelTests.cs

```diff
@@ -2,24 +2,23 @@
 {
     using System;
     using Caliburn.Micro;
+    using MessageList;
+    using MessageViewers;
+    using MessageViewers.HexViewer;
+    using MessageViewers.JsonViewer;
+    using MessageViewers.XmlViewer;
     using Microsoft.Reactive.Testing;
+    using Models;
     using NSubstitute;
     using NUnit.Framework;
     using ReactiveUI.Testing;
+    using ServiceControl;
     using ServiceInsight.Framework;
     using ServiceInsight.Framework.Events;
-    using ServiceInsight.MessageList;
-    using ServiceInsight.MessageViewers;
-    using ServiceInsight.MessageViewers.HexViewer;
-    using ServiceInsight.MessageViewers.JsonViewer;
-    using ServiceInsight.MessageViewers.XmlViewer;
-    using ServiceInsight.Models;
-    using ServiceInsight.ServiceControl;
```

Comment:
@HEskandari what is your preference? It seems that both @distantcam and I prefer full namespaces. Would you be OK with that?

Personally I prefer `using` inside `namespace` so I'd prefer to leave that as-is.